src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -84,16 +84,16 @@ limitations under the License.
 
     <command source="server" code="0x0A" name="PlaybackResponse" optional="false">
       <description>This command SHALL be generated in response to various Playback Request commands.</description>
-      <arg name="status" type="MediaPlaybackStatusEnum"/>
-      <arg name="data"   type="CHAR_STRING" optional="true"/>
+      <arg name="Status" type="MediaPlaybackStatusEnum"/>
```

Comment:
I assume `MediaPlaybackStatusEnum` is a namespacing fix for different status enums existing? Spec says just `StatusEnum`

Reply:
Yes, correct.  ZAP does not support enums in different clusters having the same name....  There are open issues on this.